src/icons/sort-desc.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="6" viewBox="0 0 12 6" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M5.59592 6L5.95569e-05 9.78411e-07L11.1918 0L5.59592 6Z" fill="black"/>
+</svg>
```

Comment:
shouldn't be desc icon on the lower part of the icon? currently it is placed same way as asc...